### DISCUSSION TITLE
Assistant / AI / Teaching

### DISCUSSION THREAD

**Discussion by selmen2004** [thonny/thonny#3365] [General]
Now that you have decided to kill assistant ( that was a great feature but little used unfortuantely )  , and use AI instead , how can we ( teachers ) give students exams / exercices  that they can solve on thonny using integrated AI ?

**Comment by aivarannamaa** [comment]
Can't you simply tell the students that the exam needs to be done without using AI?

**Comment by selmen2004** [comment]
They know of course , and wont obey too ..

**Reply by aivarannamaa** [reply]
How do you ensure they are not using ChatGPT during the exam?

**Reply by selmen2004** [reply]
mostly , they cant use chatgpt on pcs as we cut off internet during exams  , but that is not going to be a problem for sheeters as they could install ollama before exams and use it on exam ( we cant control everything ) or use their own smartphones ... 
well this is a long talk if we start it 
we got instructions that prohibit usage and installations  of any AI for teaching  ,for now,   that's why I am opening the discussion
